src/snowflake/cli/plugins/streamlit/manager.py

```diff
@@ -61,13 +62,10 @@ def _put_streamlit_files(
 
         if additional_source_files:
             for file in additional_source_files:
-                # If the file is in a folder, PUT it to the same folder in the stage
-                # If not, just PUT it to the root of the stage
-                destination = (
-                    f"{root_location}/{str(Path(file).parent)}"
-                    if "/" in file
-                    else root_location
-                )
+                if os.sep in str(file):
+                    destination = f"{root_location}/{str(file.parent)}"
```

Comment:
```suggestion
                    destination = os.path.join(root_location, str(file.parent))
```

Reply:
This is stage path and as far as I seen we use posix for those